spec/features/studio/index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe "Studio Index" do
+  before :each do 
+    @studio1 = Studio.create!(name: "Mountain Max", location: "Colorado")
+    @studio2 = Studio.create!(name: "Ocean Max", location: "California")
+    @movie1 = @studio1.movies.create!(title: "Going Hiking", creation_year: "1900", genre: "outdoor")
+    @movie2 = @studio1.movies.create!(title: "Wanda Goes Barking", creation_year: "2023", genre: "puppies")
+    @movie3 = @studio2.movies.create!(title: "Wanda VS The Vacuum", creation_year: "2022", genre: "super hero")
+    @movie4 = @studio2.movies.create!(title: "Driving I70", creation_year: "2000", genre: "action")
+  end
+
+  it "shows all studios and movies" do
+    # Story 1 - Studio Index
+
+    # As a user,
+    # When I visit the studio index page,
+    # Then I see all of the studios including name and location,
+    # And under each studio I see all of the studio's movies
+    # including the movie's title, creation year, and genre
+
+    visit "/studios"
+
+    expect(page).to have_content(@studio1.name)
+    expect(page).to have_content(@studio1.location)
+    expect(page).to have_content(@studio2.name)
+    expect(page).to have_content(@studio2.location)
+
+    expect(page).to have_content(@movie1.title)
```

Comment:
You could use within blocks to organize these tests a bit better and indicate more precisely where this information is on the page (i.e. that the movie1 attributes are in the movie1 section)

Reply:
It'd also be better to write the content assertions to expect "Genre: <genre>" and "Year: <year>" so we know you're displaying that data in the right place.